### PR TITLE
feat(client): ✨ add manifest and app icons

### LINKED ISSUE #132: Add Icons to web app
When Catniss adds a shortcut the game to her home screen it looks low resolution and it's the wrong shape. Make the icon look good to improve the perceived quality of the web app.

Tasks:
- [x] get icon from artist or use favicon image
- [x] make maskable icons
- [x] make any icons
- [x] make iOS icons
- [x] make manifest
- [x] make apple meta tags

Acceptance criteria:
- Web app has a name in the manifest
- Web app has a name in iOS
- Web app has a maskable Icon that works on Android
- Web app has a Icon with a transparent background the works on Windows
- Web app has a Icon for iOS
- Display is set to standalone in manifest (does not require service worker as part of this card)
- Display standalone or equivalent works on iOS

### PR DESCRIPTION
**Pull request by nstringham**
close #132

### CODE REVIEW

File: client/src/app.html [hunk 1]
```diff
@@ -4,6 +4,10 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%sveltekit.assets%/favicon.png" />
     <meta name="viewport" content="width=device-width" />
+
+    <title>Cat Me if You Can</title>
```
Comment (SBAsai) [inline client/src/app.html]: `if` should probably be with a capital I

File: client/src/app.html [hunk 2]
```diff
@@ -4,6 +4,10 @@
...
+    <meta name="description" content="A social deduction game for 4-8 players with cats and puns." />
```
Comment (SBAsai) [inline client/src/app.html]: This could potentially be replaced by the first paragraph in our rules modal since it's basically our game description

**Review by SBAsai — CHANGES_REQUESTED** [review]

**Review by mrjoshua520 — DISMISSED** [review]

File: client/src/app.html (hunk 1 above)
Comment (nstringham) [reply client/src/app.html]: fixed in cbb652a853949b25251a5c39c26d1ee8c1a973dc

File: client/src/app.html (hunk 2 above)
Comment (nstringham) [reply client/src/app.html]: implemented in c6ad548a0e6e80929e43d4895f29fa23d5cd4be2

**Review by mrjoshua520 — APPROVED** [review]

**Review by SBAsai — APPROVED** [review]

**Pull request merged**